automatic/dropbox/tools/chocolateyuninstall.ps1

```diff
@@ -0,0 +1,21 @@
+$packageName = 'dropbox'
+$fileType = 'exe'
+$silentArgs = '/S'
+$uninstallerPath = (getDropboxRegProps).UninstallString
+
+if ($uninstallerPath) {
+    Uninstall-ChocolateyPackage $packageName $fileType $silentArgs $uninstallerPath
+}
+
+function getDropboxRegProps() {
```

Comment:
this should be in a seperate `helpers.ps1` file, as per [1.1.8 Clean code](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#118-clean-code)